bazel_env.bzl

```
@@ -9,6 +9,9 @@ def _rlocation_path(ctx, file):
 
 def _heuristic_rlocation_path(ctx, path):
     # type: (ctx, string) -> string
+    if path.startswith("bazel-out/"):
+        path = path[path.find("external/"):]
```

This would falsely apply to paths such as `my/pkg/totally_not_external/binary` and would falsely not apply to paths such as `bazel-out/k8-fastbuild/bin/my/pkg/binary`. 

Instead, how about this:
```suggestion
        # Skip over bazel-out/<cfg>/bin.
        path = "/".join(path.split("/")[3:])
```